project-base/webpack.config.js

```diff
@@ -47,16 +47,146 @@ Encore
         }
     }))
     .addPlugin(new CopyPlugin([
-        { from: 'web/bundles/fpjsformvalidator', to: '../../assets/js/bundles/fpjsformvalidator', force: true }
+        { from: 'web/bundles/fpjsformvalidator', to: '../../assets/js/bundles/fpjsformvalidator', force: true },
+        { from: 'web/bundles/shopsysframework', to: '../../web/assets/admin/images', force: true }
     ]))
 ;
 
+// Frontend Config
+const WebfontsGenerator = require('webfonts-generator');
+const StylelintPlugin = require('stylelint-webpack-plugin');
+var requireContext = require('require-context');
+yaml = require('js-yaml');
+fs = require('fs');
+
+SVGO = require('svgo');
+const svgo = new SVGO();
+
+const svgFilesFrontend = requireContext('../../src/Resources/svg/front', false, '.svg');
+var svgFilesFrontendPath = [];
+svgFilesFrontend.keys().forEach((filePathFrontend) => {
+    var newFileFrontendPath = './src/Resources/svg/front/' + filePathFrontend;
+    svgFilesFrontendPath.push(newFileFrontendPath);
+
+    fs.readFile(newFileFrontendPath, 'utf8', function (err, data) {
+        if (err) {
+            throw err;
+        }
+        svgo.optimize(data, { path: newFileFrontendPath }).then(function (result) {
+            fs.writeFile(newFileFrontendPath, result.data, function (err) {
+                if (err) {
+                    throw err;
+                }
+                console.log('Frontend SVG icon ' + newFileFrontendPath + ' optimized');
+            });
+        });
+    });
+});
+
+const svgFilesAdmin = requireContext('../../../packages/framework/src/Resources/svg/admin', false, '.svg');
+var svgFilesAdminPath = [];
+svgFilesAdmin.keys().forEach((filePathAdmin) => {
```

Comment:
I see violation DRY principle. I see the same block of code for frontend and admin. 

And I don't really like the location. I think this call should be linked to the webpack cycle. Check out EventHooksPlugin

Reply:
I know about this DRY - I feel this part as WIP - we need to check admin paths and libs images.

Reply:
This part was refactored